src/core/markovProcessGraph.js

```diff
@@ -0,0 +1,579 @@
+// @flow
+
+/**
+ * Data structure representing a particular kind of Markov process, as
+ * kind of a middle ground between the semantic SourceCred graph (in the
+ * `core/graph` module) and a literal transition matrix. Unlike the core
+ * graph, edges in a Markov process graph are unidirectional, edge
+ * weights are raw transition probabilities (which must sum to 1) rather
```

Comment:
The "must sum to 1" constraint doesn't seem to be enforced, e.g. for an epoch node with no out-edges. In which case the edge weights sum to `(1-transitionRemainder)`.  This is based on looking inside `static new` for any re-normalization in the case where there weren't any out edges, and not finding it.